hero_chassis_controller:
  type: "hero_controllers/HeroChassisController"
  front_left_wheel: "front_left_joint"
  front_right_wheel: "front_right_joint"
  rear_left_wheel: "rear_left_joint"
  rear_right_wheel: "rear_right_joint"

  wheel_base: 0.4
  wheel_track: 0.4
  wheel_radius: 0.07625
  use_global_velocity_mode: true

  pid:
    front_left:
      p: 30.0
      i: 0.1
      d: 0.1
    front_right:
      p: 30.0
      i: 0.1
      d: 0.1
    rear_left:
      p: 30.0
      i: 0.1
      d: 0.1
    rear_right:
      p: 30.0
      i: 0.1
      d: 0.1
